test/TEST-22-TMPFILES/test-09.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Make sure that the "stat" output is not locale dependent.
+export LANG=C LC_MESSAGES=C LC_ALL=C
```

Comment:
Let's not set LC_MESSAGES if LC_ALL is set. Also, what about C.UTF-8 instead ?

Reply:
C.UTF-8 is still a Fedoraism/Debianism iirc. I mean, that's why we made it a build-time option. Hence I figured the only safe thing to do was C.

Reply:
also, i am not sure the test build scripts even install the C.UTF-8 locale in the minimal environment they put together